core/src/main/java/org/elasticsearch/search/highlight/HighlighterParseElement.java

```diff
@@ -52,16 +52,22 @@
  */
 public class HighlighterParseElement implements SearchParseElement {
 
-    private static final String[] DEFAULT_PRE_TAGS = new String[]{"<em>"};
-    private static final String[] DEFAULT_POST_TAGS = new String[]{"</em>"};
+    /** the default number of fragments for highlighting (5) */
+    private static final int DEFAULT_NUMBER_OF_FRAGMENTS = 5;
+    /** the default number of fragments size in characters (100) */
+    private static final int DEFAULT_FRAGMENT_CHAR_SIZE = 100;
```

Comment:
Same as above, I tried to limit visibility so far as long as the defaults are not needed elsewhere, but can make them public. They are constants anyway.